components/script/script_runtime.rs

```diff
@@ -163,6 +164,7 @@ pub enum ScriptThreadEventCategory {
 /// An interface for receiving ScriptMsg values in an event loop. Used for synchronous DOM
 /// APIs that need to abstract over multiple kinds of event loops (worker/main thread) with
 /// different Receiver interfaces.
+#[allow(clippy::result_unit_err)]
```

Comment:
Same as above. Can this be moved to before the `fn recv` method?

Reply:
Okay @mukilan I have updated the PR. Thank you